DSharpPlus/Entities/User/DiscordUser.cs

```diff
@@ -23,6 +23,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Dynamic;
```

Comment:
Concern.

Reply:
Unused import, not sure how that got there